Cargo.toml

```diff
@@ -36,6 +36,7 @@ glyph-names = []
 gvar-alloc = ["std"]
 
 [dev-dependencies]
-base64 = "0.13"
+base64 = "0.22.1"
 pico-args = "0.5"
+tiny-skia-path = "0.11.4"
```

Comment:
If we need it just for path transform then it can be easily implemented without any dependencies. [See this code](https://github.com/RazrFalcon/tiny-skia/blob/9b673e6c14e5e0c3f98413f8fa7e3a6cb315183b/path/src/transform.rs#L286-L290).

Reply:
Ok, you call transform invert as well... Let's leave it as is then.

Reply:
Yep... It's a dev dependency after all, so it should be fine. 😄